code/datums/status_effects/debuffs/debuffs.dm

```diff
@@ -406,7 +406,7 @@
 	return ..()
 
 /datum/status_effect/eldritch/on_apply()
-	if(owner.mob_size >= MOB_SIZE_HUMAN)
+	if(owner.mob_size >= MOB_SIZE_HUMAN || ishuman(owner)) // This is horrible but I can't see a better way to do it
```

Comment:
If anyone can come up with something better here please let me know because this hurts me physically.

Reply:
What's the problem?

Reply:
Teshari will need to be `MOB_SIZE_SMALL` in order to be picked up, and heretic marks only work on `MOB_SIZE_HUMAN` and above (otherwise heretics could get knives from/teleport to mice and other small mobs).
In order to still make the mark still affect Teshari the best thing I could come up with at 3AM was to slap an `ishuman()` check onto it, which *works*, but I'm not very happy with it as a solution.

Reply:
Eh, good enough imo.